### PR TITLE
Update MIP41c4-SP37.md + create MIP4c2-SP22.md

### PR DESCRIPTION
**Pull request by Gala-G**
Add RP URL

### CODE REVIEW

File: MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+MIP4c2-SP#: 22
+MIP to be amended: MIP41
+Author(s): Sam MacPherson (@hexonaut)
+Contributors:
+Tags: MIP41, facilitator
+Status: Withdrawn
+Date Proposed: 2022-07-03
+Date Ratified: N/A
+Forum URL: https://forum.makerdao.com/t/mip4c2-sp22-mip41-facilitator-offboarding-amendment/16297
+```
```
Comment (blimpa) [inline MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md:16]: Can you add an Extra field here?: "This proposal has been [withdrawn by its author](https://forum.makerdao.com/t/mip4c2-sp22-mip41-facilitator-offboarding-amendment/16297/36)" or something along those lines.

File: MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Amended MIPs and Components
+
+This MIP4c2 subproposal revises [MIP41](https://mips.makerdao.com/mips/details/MIP41) as well as related subcomponents [MIP41c5](https://mips.makerdao.com/mips/details/MIP41#MIP41c5).
+
+### Amendment Pull Request
+
+https://github.com/makerdao/mips/pull/603
+
+### Relevant Information
```
Comment (blimpa) [inline MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md]: Remove

File: MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md (hunk 1 above)
Comment (blimpa) [reply MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md:16]: I think adding a link to the public post is a good idea: `[withdrawn by its author](https://forum.makerdao.com/t/mip4c2-sp22-mip41-facilitator-offboarding-amendment/16297/36)`

Comment (Gala-G) [reply MIP4/MIP4c2-Subproposals/MIP4c2-SP22.md:16]: done

**Review by blimpa — APPROVED** [review]

**Pull request merged**